src/sloth256_189.cu

```diff
@@ -120,16 +138,23 @@ int main(int argc, const char *argv[])
         memset(flat, 5, sz);
         memset(flat, 3, 32);
     }
+    // below block - END
+    // here is the code inside the else block extracted as nsight safe version
+    /*
+    (cudaMallocManaged(&flat, sz));
+    memset(flat, 5, sz);
+    memset(flat, 3, 32);
```

Comment:
What is this and why is it safe and above code is not?

Reply:
> I think documentation should clearly say that it is CUDA-specific, because for regular usage all of those build instructions are irrelevant.
> Also I'm wondering if it is actually required to specify compute capability or defaults will also work as `build.rs` already does on its own.

1st one: true, will update
2nd one: it is not necessary, but it is generally considered a good practice to give this to the compiler so that it can do more optimizations if possible. User can disregard that, will update the build instructions accordingly, stating it is optional.

Reply:
It is due to `prop.integrated` or `CUDA_FATAL`. These codes are creating problems with nsight software. Nsight cannot get the profiling of the kernels with these codes are present. So that is why, this version is safe. If you are asking why they are not safe on nsight and why is this safe, I don't actually know.

Reply:
For the compute capability, it is present in the documentation that `this is an optimization and it is suggested`. So I believe it is clear that this is not mandatory. I left it as it is.